soloviova_homework1.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <math.h>
+
+int main() {
+
+  const int min_range{-50};
+  const int max_range{50};
+
+  std::cout << "Quadratic equation : Ax^2 + Bx+C =0" << std::endl;
+  std::cout << "Please, enter integers A,B,C ( in the range (-50;50)) "
+            << std::endl;
```

Comment:
```suggestion
  std::cout << "Please, enter integers A,B,C ( in the range ( " <<  min_range << ";" << max_range << ")) "
            << std::endl;
```
если вы поменяете значение переменных min_range min_range, то не забудете поменять вывод